packages/ckeditor5-table/src/tableutils.js

```diff
@@ -94,15 +94,22 @@ export default class TableUtils extends Plugin {
 	 * @param {Number} [options.columns=2] The number of columns to create.
 	 * @param {Number} [options.headingRows=0] The number of heading rows.
 	 * @param {Number} [options.headingColumns=0] The number of heading columns.
+	 * @param {String} [options.defaultProperties={}] The default properties for the created table.
+	 * @param {Object} [options.defaultCellProperties={}] The default cell properties in the created table.
```

Comment:
Also, shouldn't be just `cellAttributes`?

Reply:
Fixed in c7b23f2.